docs/conf/atlantic/2023/news/announcing-cfp.rst

```diff
@@ -0,0 +1,55 @@
+:template: {{year}}/generic.html
+
+.. post:: Mar 1, 2023
+   :tags: {{shortcode}}-{{year}}, cfp
+
+Announcing Call for Proposals
+=============================
+
+Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_.
+We'll have more updates soon,
+but we wanted to announce our CFP so you have time to prepare your talk ideas. 
+
+Speaking at Write the Docs is a fantastic way to share your ideas with our inspiring community of documentarians.
+If there is something you’d really like to see a talk on this year, submit a proposal on it, or refer someone else who would be good!
+
+While we will share some updates with a wider mailing list, sign up to the separate
+`Atlantic conference mailing list <https://www.writethedocs.org/newsletter/>`_ to stay
+up to date on all details for Write the Docs Atlantic 2023.
+
+Call for Proposals
+------------------
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+**The conference is virtual and all talks will be pre-recorded, allowing you to present and attend from anywhere!**
+Our video producer will be available to help with recording if you want it.
+We'd also love it if you'll be available for a remote live Q&A with our moderators after your talk.
+
+The schedule will run from the early afternoon to early evening in most European/African timezones,
+and from early morning to late afternoon in US East Coast/South America timezones.
+Of course, this being a virtual conference, attendees and speakers are welcomed from anywhere in the world.
+
+You can read our full `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`__ on the website.
+
+The Call for Proposals will be open until **23:59 {{cfp.ends}} {{ tz }}**.
```

Comment:
```suggestion
Read the full `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`__, 
it will be open until **23:59 {{cfp.ends}} {{ tz }}**.
```